### PR TITLE
Correctly override the equals method

### PR DESCRIPTION
**Pull request by gcmurphy**
VictimsRecord didn't correctly override the Object.equals method. This
stopped it from working correctly with things like:

```
aList.containsAll(Collection<VictimsRecord> records);
```

### CODE REVIEW

File: src/test/java/com/redhat/victims/VictimsRecordTest.java [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+	@Test
+	public void testContainsAll() throws IOException {
+		String jstr = FileUtils.readFileToString(new File(Resources.JAR_JSON))
+				.trim();
+		VictimsRecord rec1 = VictimsRecord.fromJSON(jstr);
+		VictimsRecord rec2 = VictimsRecord.fromJSON(jstr);
+
+		// Before the correct @Override this wouldn't even work..
+		ArrayList<VictimsRecord> a = new ArrayList<VictimsRecord>(); 
+		a.add(rec1);
```
Comment (abn) [inline src/test/java/com/redhat/victims/VictimsRecordTest.java:33]: Maybe add a few records into each array and also test a negative case.
